Tests/Twig/Extension/PageExtensionTest.php

```diff
@@ -41,6 +41,7 @@ public function testAjaxUrl()
 
     public function testController()
     {
+        $this->skipInPHP55();
```

Comment:
Atm there is a bug in your 3.x branch that causes your tests to fail on a specific issue with Mock objects. We already discussed a lot about it, since these test have nothing to do with my issue that i solved.